README.md

```diff
@@ -111,6 +111,69 @@ describe('application launch', function () {
 })
 ```
 
+### With AVA
+
+Spectron also works with [AVA](https://github.com/sindresorhus/ava) that allows you to write your test code in ES2015 without extra support.
+
+```js
+'use strict';
+
+import test from 'ava';
```

Comment:
Is babel required to use this syntax or is there a version of node that supports the `import` syntax?